### PR TITLE
[NUXE-212] KIT DEV- Form Group Kit

### PR DESCRIPTION
**Pull request by adub65**
#### Screens
Screen Shot 2020-11-16 at 6 03 01 PM
Screen Shot 2020-11-16 at 6 03 46 PM
Screen Shot 2020-11-16 at 6 03 15 PM
Screen Shot 2020-11-16 at 6 03 57 PM
Screen Shot 2020-11-16 at 6 03 24 PM
Screen Shot 2020-11-16 at 6 04 07 PM

#### Breaking Changes

no, it's a new kit

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUXE-212
https://github.com/powerhome/playbook/issues/1101

#### How to test this
 run rspec

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

**Comment by thestephenmarshall** [comment]
@adub65 - looks like this one needs to have `v7.7.0` merged into it. This will likely result in some conflicts, so be sure to reach out if you need extra eyes on it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 
 
+## 7.4.1
+
+**Fixed Bugs:**
+
+- Revert “Timestamp Kit Formatted Date Additions” [#1150](https://github.com/powerhome/playbook/pull/1150)  [#1180](https://github.com/powerhome/playbook/pull/1180) 
+
```
Comment (thestephenmarshall) [inline CHANGELOG.md:8]: This change shouldn't be here.

File: Gemfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    playbook_ui (7.4.0)
+    playbook_ui (7.4.1)
```
Comment (thestephenmarshall) [inline Gemfile.lock:4]: `7.6.2` is the latest branch

**Review by bh247484 — COMMENTED** [review]
This looks good barring alpha testing results.  Removed all unrelated file changes.

**Comment by thestephenmarshall** [comment]
@adub65 on the Form Group React Typeahead example, I am seeing this:

Screen Shot 2020-11-11 at 10 08 56 AM

On the Rails version, it looks like your width is working but I get no results:

Screen Shot 2020-11-11 at 10 10 34 AM

**Review by thestephenmarshall — CHANGES_REQUESTED** [review]
I think there is a little more tweaking for the React Typeahead example. Looks like the width isn't working there. This is likely bc the React version uses the [react-select](https://react-select.com) lib.

File: playbook/app/pb_kits/playbook/pb_form_group/_form_group.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+
+  & > [class^=pb_selectable_card_icon_kit]:not(:last-child) {
+    [class^=pb_selectable_card_kit] input[type="checkbox"]:not(:checked) ~ label, [class^=pb_selectable_card_kit] input[type="radio"]:not(:checked)~ label {
+      border-right-width: 0;
+    }
+  }
+
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_form_group/_form_group.scss:99]:
```suggestion
& > [class^=pb_typeahead_kit] {
  min-width: 200px;
}
```

I would suggest adding this to fix the issue with the React version of Typeahead example being too narrow. 

Screen Shot 2020-11-11 at 10 26 16 AM

**Review by thestephenmarshall — APPROVED** [review]

**Review by thestephenmarshall — CHANGES_REQUESTED** [review]
On the Rails version of the Typeahead group, you will want to add the inline script tag which handles the selection of options or else the doc for that example will not allow you to choose anything. Let me know if you have any questions.

**Review by thestephenmarshall — APPROVED** [review]
@adub65 Nice work! Let's make Typeahead examples into a new ticket to work separately given there are concerns around color and multi-line. cc @jasoncypret

**Pull request merged**